utils/haptics.ts

```diff
@@ -0,0 +1,148 @@
+import * as ExpoHaptics from "expo-haptics";
+import { runOnJS } from "react-native-reanimated";
+
+import { sentryTrackError } from "./sentry";
+
+const impactFeedbackStyle = ExpoHaptics.ImpactFeedbackStyle;
+
+const notificationFeedbackStyles = ExpoHaptics.NotificationFeedbackType;
+
+const selectionAsync = () => {
+  try {
+    // logger.debug("selectionAsync");
+    ExpoHaptics.impactAsync(impactFeedbackStyle.Light).catch(sentryTrackError); // Selection doens't seem to work? So use light impact for now
+  } catch (error) {
+    sentryTrackError(error);
```

Comment:
Goes for all of these, is there a world where this would ever get hit? Or just planning for the future if additional things get added in here?

Reply:
you're right it's overkill. removed!